### DISCUSSION TITLE
Is it possible to turn off power button light?

### DISCUSSION THREAD

**Discussion by pripishchik** [BartoszCichecki/LenovoLegionToolkit#722] [Q&A] [answered]
I would like to completely turn off power button light on my Lenovo Legion 7 (2022). Is it possible?

**Comment by BartoszCichecki** [comment, marked as answer]
As far as I know it is not possible.

**Reply by pripishchik** [reply]
what a pity, looks like it is possible on ThinkPad via this software https://github.com/valinet/ThinkPadLEDControl
